src/app/routing/root-router.component.tsx

```diff
@@ -1,11 +1,13 @@
 import { NavigationContainer } from '@react-navigation/native';
 import React from 'react';
+import { useSelector } from 'react-redux';
 
 import { AuthStack } from '../authentication/routing';
+import { getIsLoggedIn } from '../authentication/selectors';
 
 import { MainStackComponent } from './main-stack.component';
 
 export const RootRouterComponent: React.FC = () => {
-    const user = 'user';
-    return <NavigationContainer>{user ? <MainStackComponent /> : <AuthStack />}</NavigationContainer>;
+    const isLoggedIn = useSelector(getIsLoggedIn);
+    return <NavigationContainer>{!isLoggedIn ? <MainStackComponent /> : <AuthStack />}</NavigationContainer>;
```

Comment:
We need to show MainStack if isLoggedIn === true, so remove ! symbol

Reply:
That was just to test login screen